docs/source/audio_dataset_script.mdx

```diff
@@ -0,0 +1,337 @@
+# Create an audio dataset
+
+There are two methods for creating and sharing an audio dataset. This guide will show you how to:
+
+* Create an audio dataset with `AudioFolder` and some metadata. This is a no-code solution for quickly creating an audio dataset.
+
+* Create an audio dataset by writing a loading script. This method is a bit more involved, but you have greater flexibility over how a dataset is defined, downloaded, and generated.
+
+<Tip>
+
+You can control access to your dataset by requiring users to share their contact information first. Check out the [Gated datasets](https://huggingface.co/docs/hub/datasets-gated) guide for more information about how to enable this feature on the Hub.
+
+</Tip>
+
+## AudioFolder
+
+The `AudioFolder` is a dataset builder designed to quickly load an audio dataset without requiring you to write any code. `AudioFolder` automatically infers the class labels of your dataset based on the directory name. Just store your dataset in a directory structure like:
```

Comment:
As suggested by @mariosasko [here](https://github.com/huggingface/datasets/pull/4783#issuecomment-1230159561), I also think it makes more sense to have the sections explaining how to create a dataset for a specific task in this doc instead of `audio_load.mdx`. Let me know what you think! :)

Reply:
I think it's a good idea to describe it in more detail in a separate section indeed, thanks!
however, the difference between ImageFolder and AudiFolder usage is that I assume that for audio folder the most common task is automatic speech recognition which requires metadata, so to me it makes sense to describe first this use case and then make a note about a classification task and automatically inferred labels.